lib/commands/fleets/index.ts

```diff
@@ -77,17 +77,29 @@ export default class FleetsCmd extends Command {
 			application.device_type = application.is_for__device_type[0].slug;
 		});
 
-		await this.outputData(
-			applications,
-			[
+		const applicationsToDisplay = applications.map((application) => ({
+			id: application.id,
+			app_name: application.app_name,
+			slug: application.slug,
+			device_type: application.device_type,
+			online_devices: application.online_devices,
+			device_count: application.device_count,
+		}));
+
+		if (options.json) {
+			console.log(JSON.stringify(applicationsToDisplay, null, 4));
+			return;
+		}
+
+		console.log(
+			getVisuals().table.horizontal(applicationsToDisplay, [
 				'id',
-				'app_name',
+				'app_name => NAME',
 				'slug',
 				'device_type',
-				'device_count',
 				'online_devices',
-			],
-			options,
+				'device_count',
```

Comment:
Any reason that you moved this last, which in the json it is in the reverse order?
In the way it was before it was also more consistent w/ the order that we have those counts in the dashboard as well, which I'm guessing is where we got the idea to add `online_devices` to the cli as wel.